ckan/public/base/javascript/modules/image-upload.js

```diff
@@ -38,20 +40,23 @@ this.ckan.module('image-upload', function($, _) {
 
       // firstly setup the fields
       var field_upload = 'input[name="' + options.field_upload + '"]';
-      var field_url = 'input[name="' + options.field_url + '"]';
-      var field_clear = 'input[name="' + options.field_clear + '"]';
-      var field_name = 'input[name="' + options.field_name + '"]';
+      var field_url    = 'input[name="' + options.field_url + '"]';
+      var field_clear  = 'input[name="' + options.field_clear + '"]';
+      var field_name   = 'input[name="' + options.field_name + '"]';
```

Comment:
I am in favor of not adding additional whitespace here as it makes the changes seem much bigger then they actually are. It also seems like aligning equal signs is not done in the other parts of the javascript so I would not change that here.